docs/deployment/getting_started_locally.md

```diff
@@ -118,6 +152,13 @@ kubectl --kubeconfig=/tmp/kubeconfig-shoot-local.yaml get nodes
 make kind-down
 ```
 
+## (Optional): Tear down the Gardener environment if you added a second seed
+
+``` shell
+make kind2-down
+make kind-down
```

Comment:
`make kind-down` should not be needed again here (it was already mentioned in L152)?